config/custom_items.txt

```diff
@@ -1399,4 +1399,25 @@ item_desc: A customization kit with all the parts needed to convert a RIG.
 kit_name: ADM.i.RE RIG
 kit_desc: A worn, tempered RIG-suit with heavy plating and a custom breathing apparatus. Looks like it's been through a lot of exponentially heightened power, and seems capable of staking government land with a geiger counter in its hand.
 kit_icon: admirerig
+}
+
+{
+ckey: FalseShepardPie
+character_name: SINA-499
+item_path: /obj/item/clothing/head
+item_name: Corsair's Tricorn
+item_desc: A tricorn hat with gold trim. Yarr.
+item_icon: sina_499_hat
+}
+
+{
+ckey: FalseShepardPie
+character_name: SINA-499
+item_path: /obj/item/kit
+item_name: Corsair's Plate Carrier
+item_desc: A customisation kit with all the parts needed to convert a plate carrier.
+kit_name: corsair's plate kit
```

Comment:
You've mixed up the kit and the item. In this case, the kit is the item and the kit name is what it's changing when used.